crates/runner/src/lib.rs

```diff
@@ -27,6 +28,8 @@ pub struct Builder {
     built: bool,
     /// Preload the module at path, using the given instance name.
     preload: Option<(String, PathBuf)>,
+    /// Whether to use the `compile` or `build` command.
+    use_compile: bool,
```

Comment:
Could we make this an enum instead? (i.e., `command: JavyCommand`) I understand that this is probably temporary, until the `compile` command is fully deprecated, however it'll probably be easier to extend/refactor later by chasing down an enum rather than a boolean.